build/recipes.mk

```diff
@@ -3,7 +3,7 @@
 # Makefile fragment for recipe actions
 #
 EMBEDDED_RECIPES_PATH         ?= internal/install/recipes/files
-RECIPE_ARCHIVE_VERSION        ?= $(shell curl -Ls https://download.newrelic.com/install/open-install-library/currentVersion.txt)
+RECIPE_ARCHIVE_VERSION        ?= $(shell curl -Ls https://s3.us-east-1.amazonaws.com/nr-downloads-main/install/open-install-library/currentVersion.txt)
```

Comment:
out of curiosity, what would happen to dev users building when we move our download to another s3 bucket or region? Would it be obvious that this is the place to update?

Reply:
This is the only place to update, correct.

Reply:
I think so